backend/src/main/resources/db/migration/V0025__move_colors_to_problem.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE problem ADD COLUMN `hold_color_primary` VARCHAR(7) NOT NULL AFTER number;
+ALTER TABLE problem ADD COLUMN `hold_color_secondary` VARCHAR(7) NULL DEFAULT NULL AFTER hold_color_primary;
+
+UPDATE problem
+    JOIN color ON problem.color_id = color.id
+    SET hold_color_primary = color.rgb_primary, hold_color_secondary = color.rgb_secondary;
```

Comment:
Re-run on prod data to double check that dual color problems are properly migrated.